### PR TITLE
Form add

### PR DESCRIPTION
**Pull request by NomanRamzan**
Add the form section and validation with Partner Contribution

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -258,5 +258,70 @@ <h2>I’am a Software Developer</h2>
...
+        <div>
+          <textarea
+            name="user_Message"
+            class="msg-area"
+            placeholder="Enter text here"
+            maxlength="500"
+          ></textarea>
+        </div>
+        <div>
+          <button class="form-button">Get in touch</button>
```
Comment (ShahierNashaat) [inline index.html:303]: - [ ] Please add type="submit" to your button to make the form work.

File: index.html [hunk 2]
```diff
@@ -258,5 +258,70 @@ <h2>I’am a Software Developer</h2>
...
+            required
+          />
+        </div>
+        <div>
+          <textarea
+            name="user_Message"
+            class="msg-area"
+            placeholder="Enter text here"
+            maxlength="500"
+          ></textarea>
```
Comment (ShahierNashaat) [inline index.html:300]: - [ ] Please be noticed that the textarea must have a required attribute to stop submitting the form without filling it.

File: index.html [hunk 3]
```diff
@@ -258,5 +258,70 @@ <h2>I’am a Software Developer</h2>
...
+          </li>
+          <li class="item-footer">
+            <img src="/images/in-black.png" alt="in" />
+          </li>
+          <li class="item-footer">
+            <img src="/images/hand-black.png" alt="hand" />
+          </li>
+          <li class="item-footer">
+            <img src="/images/twitter-black.png" alt="twitter" />
+          </li>
```
Comment (ShahierNashaat) [inline index.html:320]: - [ ] Please start the path of your images with `./` instead of `/` example `github`

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @NomanRamzan and @haftamudesta , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @NomanRamzan and @haftamudesta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**